rasa/core/train.py

```diff
@@ -107,7 +108,7 @@
                 )
 
                 with TempDirectoryPath(tempfile.mkdtemp()) as train_path:
-                    await train(
+                    train_coroutine = train(
```

Comment:
Is this the sort of typing you always get at me for? 😄 (i forget the name)

Reply:
hungarian notation 😀You're completely right. I assigned a variable to not chain too many calls in `asyncio.gather`. But it's actually not to bad, moved it back in.